### PR TITLE
Chore/brand refresh/update min sdk

### PR DESCRIPTION
**Pull request by HarmonyM-GC**
Recent brand [refresh splash screen](https://github.com/GooseChase/goosechase_android/pull/160) updates required a SDK bump. 

This PR updates gradle wrapper and properties, compile sdk, target sdk, min sdk. It also fix some manifest errors that are require by the new versions. 

I have tested this on 2 devices: Samsung Galaxy S8 and Samsung Galaxy S21 Ultra 5G both the demo app and the goosechase image capture, video capture, video trim function as expected. 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202451828225400

### CODE REVIEW

File: demo/build.gradle [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
         applicationId "com.flurgle.camerakit.demo"
-        minSdkVersion 15
-        targetSdkVersion 25
+        minSdkVersion 21
+        targetSdkVersion 33
         versionCode 4
         versionName "1.0.3"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        multiDexEnabled true
```
Comment (agodding) [inline demo/build.gradle]: Are we using multidex? 😱

File: gradlew [hunk 2]
```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```
Comment (agodding) [inline gradlew:1]: Was this update to gradlew on purpose?

**Review by agodding — APPROVED** [review]
Curious if the gradle wrapper updates are on purpose and/or necessary. Otherwise LGTM if it's tested and works!

File: gradlew (hunk 2 above)
Comment (HarmonyM-GC) [reply gradlew:1]: These were all automatic gradle updates that AS handled.

File: demo/build.gradle (hunk 1 above)
Comment (HarmonyM-GC) [reply demo/build.gradle]: Let me double check this. It might have slipped in the copy past with the vectorDrawables.useSupportLibrary = true

Comment (HarmonyM-GC) [reply demo/build.gradle]: It was not needed for the camerakit - I've tested and removed.

**Pull request merged**